benchmark_results.json

```diff
@@ -0,0 +1 @@
+{"function_name": "search", "start_time": "2023-10-18T17:18:21.563478", "finish_time": "2023-10-18T17:18:21.935885"}
```

Comment:
I don't think we should be checking in benchmark results.

Reply:
also: you'll need to differentiate the version of the search you are testing, otherwise how will you compare before and after? MicroBench supports additional key/values:

https://pypi.org/project/microbench/

"Note that keyword arguments can be supplied to the constructor (in this case some_info=123) to specify additional information to capture"

```benchmark = MyBench(some_info=123)```

Reply:
As discussed I will add git commit, FunctionCall, ReturnValue and version of the function. 

Only git commit has been added for now.

Reply:
I would like to see an example of the output since we decided to checkin the benchmark results. don't forget to move benchmarks to subdir with filename containing function name tested.

Reply:
done